### PR TITLE
Fix naming, typo, ... fix ccmq value for AES auth ccm mode

### PR DESCRIPTION
**Pull request by e-puerto**

### CODE REVIEW

File: src/blockcipher/aes/scl_aes_ccm.c [hunk 1]
```diff
@@ -67,36 +67,64 @@ int32_t scl_aes_ccm_init(const metal_scl_t *const scl_ctx, aes_auth_ctx_t *const
...
-        // 2¹⁶
-        if (pld_byte_len < 0x10000) 
+        // 2^24
+        if ( pld_byte_len < (((uint64_t)1 << (3*8))) )
         {
-            ccmq = 2;
+            ccmq = 3;
         } 
         else
+        // 2^32
```
Comment (phmoussay) [inline src/blockcipher/aes/scl_aes_ccm.c:94]: Can we use if else statement instead of if imbrication.

File: src/blockcipher/aes/scl_aes_ccm.c [hunk 2]
```diff
@@ -67,36 +67,64 @@ int32_t scl_aes_ccm_init(const metal_scl_t *const scl_ctx, aes_auth_ctx_t *const
...
+                    {
+                        if ( pld_byte_len < (((uint64_t)1 << (7*8))) )
+                        {
+                            ccmq = 7;
+                        } 
+                        else
+                        // 2^64
+                        {
+                            ccmq = 8;
+                        }
```
Comment (phmoussay) [inline src/blockcipher/aes/scl_aes_ccm.c]: Can we use else if statement instead of if imbrication.

Comment (e-puerto) [reply src/blockcipher/aes/scl_aes_ccm.c]: Changed

**Review by phmoussay — APPROVED** [review]
LGTM

**Pull request merged**